### DISCUSSION TITLE
[QUESTION] Why this magic comment `!TEX options` stops building?

### DISCUSSION THREAD

**Discussion by adhxtry** [James-Yu/LaTeX-Workshop#3097] [Q&A]
I am so sorry about creating a new issue tho alredy had created one at past #2997 but I created the issue as a bug report which was a wrong decision. Anyways here is the problem I am facing:

```tex
% !TEX program = pdflatex
% !TEX options = -shell-escape %DOC%.tex

\documentclass[
  convert={convertexe={magick},density=1200,outext=.png}
  ]{standalone}
\newenvironment{mymath}{$\displaystyle}{$}

\begin{document}

\begin{mymath}
    \sum_{n=0}^{100}\underbrace{e^{f'(\sqrt n)}}
\end{mymath}

\end{document}
```

The above tex should be executed like `pdflatex -shell-escape sample.tex` which generates a png image. In the #2997 I was asked to add `-halt-on-error -interaction=nonstopmode` that means `% !TEX options = -halt-on-error -interaction=nonstopmode -shell-escape %DOC%.tex` or something? When I save the file it should update the png which it doesn't rather it gets stuck in the building. Sorry I wasn't clear in the last issue. Please help me find the mistake. Thanks 😃

**Comment by tamuratak** [comment]
>  that means % !TEX options = -halt-on-error -interaction=nonstopmode -shell-escape %DOC%.tex or something? 

Yes.

> hen I save the file it should update the png which it doesn't rather it gets stuck in the building.

This issue is not related to LaTeX Workshop. We cannot help you. Please find help somewhere else.
